cmd/cmd.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
```

Comment:
what are your thoughts about cobra vs kingpin?

Reply:
https://github.com/alecthomas/kingpin last commit 9 month ago, either it's perfect or dead. I'll check if we can use it.